app/src/main/java/org/coepi/android/ui/onboarding/OnboardingFragment.kt

```diff
@@ -19,4 +22,10 @@ class OnboardingFragment : Fragment() {
         vm = viewModel
     }.root
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val textView = view!!.findViewById<TextView>(R.id.onboardingLink)
```

Comment:
Use view binding. Look e.g. in `SymptomsFragment`. Also, _never_ use `!!` (here it will be solved by using view binding).

Reply:
Updated to use view binding and remove the !! operator.